src/lib/util/Timestamp.js

```diff
@@ -1,5 +1,119 @@
 const { TIME: { SECOND, MINUTE, DAY, DAYS, MONTHS, TIMESTAMP: { TOKENS } } } = require('./constants');
 
+/* eslint-disable id-length, new-cap */
+
+// Dates
+
+const Y = time => String(time.getFullYear()).slice(0, 2);
```

Comment:
You are declaring a bunch of variables to use exactly once...

Reply:
Expanded and inlined all the code 👍